tests/stubdata/input/jats_pnas_1715554115.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've been cutting the `body` section out of these input files so we don't make any publishers mad. We don't need the body for our purposes, so let's remove it.

Reply:
Oh, not related, but I also just noticed that something is up with one of the unittests - I'm guessing you saw it, but if not, just fyi.